.gitignore

```diff
@@ -20,3 +20,6 @@ tmp
 mkmf.log
 target
 ruby/ext/rust_example/Makefile
+ruby/ext/rust_example/rust_example.o
+ruby/ext/rust_example/rust_example.so
+ruby/ext/rust_example/librust_example.a
```

Comment:
True. The only reason I deleted them is that I was rebuilding and then accidentally committed the changed files.